File: pr-jpa/src/main/java/com/example/jpa/domain/dao/Foo.java
package com.example.jpa.domain.dao;

import lombok.Data;

import javax.persistence.*;
import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "foo")
public class Foo {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String bar1;
    private String bar2;
    private String bar3;

    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: kotlin-boilerplate/kotlin-domain/src/main/kotlin/domain/dao/test/Foo.kt
package domain.dao.test

import java.time.LocalDateTime
import javax.persistence.*

@Entity
@Table(name = "foo")
class Foo (
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  var id: Long = Long.MIN_VALUE,
  var bar1: String,
  var bar2: String,
  var bar3: String,
  @Column(name = "created_at")
  var createdAt: LocalDateTime,
  @Column(name = "updated_at")
  var modifiedAt: LocalDateTime
)
